ui_framework/doc_site/src/components/guide_page/guide_page_container.js

```diff
@@ -0,0 +1,10 @@
+import { connect } from 'react-redux';
+import { GuidePage } from './guide_page.jsx';
+
+function mapStateToProps(state, ownProps) {
+  return Object.assign({}, ownProps, {
```

Comment:
What's the best practice here? Should I assign the properties to `ownProps` instead?